client/coral-framework/reducers/config.js

```diff
@@ -1,30 +1,28 @@
-/* @flow */
-
 import {Map} from 'immutable';
-import * as actions from '../actions/config';
+import * as actions from '../constants/config';
```

Comment:
This makes sense as a standard, but it's not reflected for items and notifications. Should we make that update here or put it in a separate PR? If we're going to refactor we should make it consistent throughout.